android/src/com/mapswithme/maps/editor/data/NamesDataSource.java

```diff
@@ -0,0 +1,27 @@
+package com.mapswithme.maps.editor.data;
+
+/**
+ * Class which contains array of loclized names with following priority:
+ *  1.Names for Mwm languages
+ *  2.User`s language name
+ *  3.International name
+ *  4.Other names
+ * and mandatoryNamesCount - count of names which should be always shown.
+ */
+public class NamesDataSource {
+        private LocalizedName[] mNames;
+        private int mMandatoryNamesCount;
+
+        public NamesDataSource(LocalizedName[] names, int mandatoryNamesCount) {
+            this.mNames = names;
+            this.mMandatoryNamesCount = mandatoryNamesCount;
+        }
+
+        public LocalizedName[] getNames() {
```

Comment:
Why not to just make both fields public? And to add a final specificator to mMandatoryNamesCount.